File: leetcode/longestcommonPrefix.js
var longestCommonPrefix = function(strs) {
  let prefix = '';
    if (Array.isArray(strs)){
        strs[0].split('').some(str=>{
            let s = str;
            if(s.length >0 && strs.every(v=> v.includes(s))){
                prefix += s;
            }else{
              return true;
            }
        })
    }
    return prefix;
};

File: leetcode/longestcommonPrefix.cpp
string longestCommonPrefix(vector<string>& strs) {
        string ans = "";
        for (int j=0;j<strs[0].size();j++){
            char tmp = strs[0][j];
            for (int i =1;i<strs.size();i++){
                if (tmp != strs[i][j]) return ans;
            }
            ans += tmp;
        }
        return ans;
        
    }
